### PR TITLE
OE-679 Settlement Report dev guide

### PR DESCRIPTION
**Pull request by pzelnip**
Can see a preview at: http://dev.beanstream.com.oe-679.s3-website-us-east-1.amazonaws.com/docs/guides/settlement/

### CODE REVIEW

File: source/docs/guides/settlement.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
```
Comment (david-light) [inline source/docs/guides/settlement.md]: Should url be settlement_report to keep it consistent with other guides and not as a guide on how settlement works?

File: source/docs/guides/index.md [hunk 2]
```diff
@@ -89,6 +89,12 @@ cards:
             Schedule series of automatic payments.
         icon: creditcard-outline
         link: /docs/guides/recurring_payment
+    -
+        title: Settlement Report
```
Comment (david-light) [inline source/docs/guides/index.md:93]: This needs to be added to the left menu on the Guides page
http://dev.beanstream.com.oe-679.s3-website-us-east-1.amazonaws.com/docs/guides/

File: source/docs/guides/settlement.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+pending settlement, settlement amount, when to expect it, and what fees were
+removed. With the settlement data, you can spend less time on manual tasks and
+reconciliation.
+
+As a Bambora Partner, you can help your customers reconcile by
+displaying settlement data within your software by using the Settlement Report.
+It allows you to display to your customers their pending settlement, settlement
+amount, when they can expect it, and what fees were removed. With the settlement
+data, your customers can spend less time on manual tasks and reconciliation.
+
```
Comment (david-light) [inline source/docs/guides/settlement.md]: This seems to repeat the above. What about separating out the shared and the as a user parts (which I like)

The settlement report allows you to display to your customers their pending settlement, settlement
amount, when they can expect it, and what fees were removed. With the settlement
data, you and your customers can spend less time on manual tasks and reconciliation.

As a Bambora Merchant, the Settlement Report provides an overview of
transactions and upcoming settlement disbursement.

As a Bambora Partner, you can help your customers reconcile by
displaying settlement data within your software by using the Settlement Report.

File: source/docs/guides/settlement.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+the endpoint: `start_date` and `end_date`, both of which are supplied as query
+string parameters on the HTTP `GET` request.
+
+```no-highlight
+Note: Both of these values are required on all requests to the Settlement Report API
+```
+
+Both parameters accept a date, and will restrict settlement records to the date
+range specified (inclusive).  More details are in the formal specification at
+<https://dev.na.bambora.com/docs/references/payment_APIs/v1-0-5/#smooth-scroll-top>
```
Comment (david-light) [inline source/docs/guides/settlement.md]: Do you need smooth-scroll-top?

**Review by david-light — CHANGES_REQUESTED** [review]

File: source/docs/guides/settlement.md (hunk 1 above)
Comment (pzelnip) [reply source/docs/guides/settlement.md]: Good suggestion, done.

File: source/docs/guides/settlement.md (hunk 4 above)
Comment (pzelnip) [reply source/docs/guides/settlement.md]: Nope, removed.

File: source/docs/guides/index.md (hunk 2 above)
Comment (pzelnip) [reply source/docs/guides/index.md:93]: Nice catch, fixed.

File: source/docs/guides/settlement.md (hunk 3 above)
Comment (pzelnip) [reply source/docs/guides/settlement.md]: :shrug: That copy is entirely Matt's words.

**Review by dan-sparkes-bambora — APPROVED** [review]

**Pull request merged**